R/template-utils.R

```diff
@@ -0,0 +1,47 @@
+#' Generate display string
+#' 
+#' Populates passed template with values from display_variables after verifying.
+#' 
+#' @param template string template containg variables to insert surrounded by {}
+#' @param display_variables named list with values to insert where name is equal to name in template
+#' @param ID string containing ID used in warning generation
```

Comment:
Used when disolaying warnings in case of errors in the input. This should allow the end user to easily know which entity in the dictionary has problematic values.